### DISCUSSION TITLE
Tooltip text

### DISCUSSION THREAD

**Discussion by Dregi56** [indomus/forum#1467] [HUB: Home Assistant]
Qualcuno mi sa suggerire come può essere impostato il text che compare come tooltip quando si passa il mouse sopra una voce della scheda lovelance? Di solito un tooltip dovrebbe essere "esplicativo" , ma attualmente ripete la stessa voce su cui si va in _onmouseover_

**Comment by Larry-78** [comment]
Ciao, se installi [button-card]( https://github.com/custom-cards/button-card), potresti utilizzare l'opzione `tooltip` ovviamente non è disponibile per touchscreen ma è supportata dai [templates](https://github.com/custom-cards/button-card#javascript-templates). È possibile configurare un `tooltip` e sarà visibile per 1.5s
